src/views/v2/rsvp/messages/must-login.php

```diff
@@ -40,7 +40,7 @@
 			?>
 
 			<a
-				href="<?php echo esc_url( $login_url . '?tribe-tickets__rsvp' . $rsvp->ID ); ?>"
+				href="<?php echo esc_url( $login_url . '?tribe-tickets__rsvp-' . $rsvp->ID ); ?>"
```

Comment:
Should this be `?tribe-tickets__rsvp=`?

Reply:
No, it's actually not even working or changing anything. I took it from the current views, I feel that was there to have an anchor so when the login is completed it returns to the event page and scrolls there.

Reply:
An anchor would be # should we use that?

Reply:
Right, the thing is that the ID should be in place as well, and it's not. Probably I'll revisit this when we implement the functionality. The idea of leaving it was to come to this again when implementing the functionality.